framework/framework.go

```diff
@@ -232,22 +191,21 @@ func (f *Framework) DeleteFunc(obj interface{}) {
 	f.mutex.Lock()
 	defer f.mutex.Unlock()
 
-	ctx := context.Background()
-	ctx, err := f.initCtxFunc(ctx, obj)
+	resourceSet, err := f.resourceRouter.ResourceSet(obj)
 	if err != nil {
-		f.logger.LogCtx(ctx, "error", fmt.Sprintf("%#v", err), "event", "delete")
+		f.logger.Log("error", fmt.Sprintf("%#v", err), "event", "delete")
```

Comment:
This one should be LogCtx IMO

Reply:
You may notice that there is no context yet.

Reply:
lol. This is very true :D